rosa/cli.py

```diff
@@ -45,21 +47,26 @@ def is_logged_in(aws_region=None, allowed_commands=None):
         res = build_execute_command(
             command="whoami", aws_region=aws_region, allowed_commands=_allowed_commands
         )
-        return "User is not logged in to OCM" not in res["err"]
+        return "User is not logged in to OCM" not in res
     except CommandExecuteError:
         return False
 
 
-def execute_command(command):
+def execute_command(command, wait_timeout=TIMEOUT_5MIN):
     joined_command = " ".join(command)
     LOGGER.info(
-        f"Executing command: {re.sub(r'(--token=.* |--token=.*)', '--token=hashed-token ', joined_command)}"
+        f"Executing command: {re.sub(r'(--token=.* |--token=.*)', '--token=hashed-token', joined_command)}, "
+        f"waiting for {wait_timeout} seconds."
     )
-    res = subprocess.run(command, capture_output=True, text=True)
-    if res.returncode != 0:
-        raise CommandExecuteError(f"Failed to execute: {res.stderr}")
-
-    return parse_json_response(response=res)
+    child = pexpect.spawn(joined_command, encoding="utf-8", timeout=wait_timeout)
```

Comment:
`pexpect` use to interact with shell, I don't see here any interaction with shell output